docs/man_pages/publishing/publish-remove.md

```diff
@@ -0,0 +1,19 @@
+publish remove
```

Comment:
publish remove
==========

Usage \| Synopsis
------\|-------
General \| \`$ appbuilder publish remove [\<Connection ID\>]\`

Removes a saved connection. If you do not provide one or more command parameters, the Telerik AppBuilder CLI shows an interactive prompt to let you set the remaining details. 

\#\#\# Attributes
* \`\<Connection ID\>\` is the index or name of the connection as listed by \`$ appbuilder publish`

\<% if(isHtml) { %\>
\#\#\# Related Commands

Command \| Description
----------\|----------
\[publish\]\(publish.html\) \| Lists all saved connections or publishes to the selected connection.
\[publish add\]\(publish-add.html\) \| Saves a new server connection.
\<% } %\>